ResourceManager/GitExtensionsControl.cs

```diff
@@ -69,15 +69,22 @@ public virtual bool ProcessHotkey(Keys keyData)
         /// <param name="hotkeySettingsName">The setting name.</param>
         protected void LoadHotkeys(string hotkeySettingsName)
         {
-            _hotkeys = null;
+            _hotkeys = GetHotkeys(hotkeySettingsName);
+        }
 
+        /// <summary>
+        ///  Get hotkeys for the specified configuration setting.
+        /// </summary>
+        /// <param name="hotkeySettingsName">The setting name.</param>
+        protected IReadOnlyList<HotkeyCommand>? GetHotkeys(string hotkeySettingsName)
+        {
             if (!HotkeysEnabled || !_serviceProviderLoaded)
             {
                 // Hotkeys shall be loaded by all controls in OnRuntimeLoad
-                return;
+                return null;
             }
```

Comment:
Rather `return []` in order to avoid nullable IReadOnlyList, which will simplify all usages.